runtime/tools/python/ttrt/__init__.py

```diff
@@ -3,18 +3,27 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import os
-import json
 import importlib.machinery
-import sys
-import signal
-import os
-import io
-import subprocess
-import time
-import socket
-from pkg_resources import get_distribution
-import sys
-import shutil
+import importlib.util
+
+
+def get_ttrt_metal_home_path():
+    package_name = "ttrt"
+    spec = importlib.util.find_spec(package_name)
+    package_path = os.path.dirname(spec.origin)
+    tt_metal_home = f"{package_path}/runtime"
+    return tt_metal_home
+
+
+os.environ["TT_METAL_HOME"] = get_ttrt_metal_home_path()
+
+new_linker_path = f"{get_ttrt_metal_home_path()}/tests"
+current_ld_library_path = os.environ.get("LD_LIBRARY_PATH", "")
+if current_ld_library_path:
+    updated_ld_library_path = f"{new_linker_path}:{current_ld_library_path}"
+else:
+    updated_ld_library_path = new_linker_path
+os.environ["LD_LIBRARY_PATH"] = updated_ld_library_path
```

Comment:
change is good enough for me. I think it might be worth adding in something similar to how tt-metal does it


`third_party/tt-metal/src/tt-metal/ttnn/ttnn/library_tweaks.py`

they import this in `third_party/tt-metal/src/tt-metal/ttnn/ttnn/__init__.py`

```
import ttnn.library_tweaks

library_tweaks.setup_ttnn_so()
```

maybe we can do something similar so our __init__.py stays more clean. something like adding another file in python/ttrt called library_tweaks. Up to you if you think this would be a useful change so others don't trip on this in the future.

Reply:
thank you!